src/styles/StyleUtils.js

```diff
@@ -448,6 +448,29 @@ function getPaddingLeft(paddingLeft) {
     };
 }
 
+/**
+ * Generate a style for the ImageCropView, based on its container size.
+ * @param {Number} containerSize
+ * @returns {Object}
+ */
+function getImageCropViewStyle(containerSize) {
+    return {
+        height: containerSize,
+        width: containerSize,
+    };
+}
+
+/**
+ * Generate a style for the Slider, based on its container width.
+ * @param {Number} sliderContainerSize
+ * @returns {Object}
+ */
+function getAvatarCropSliderStyle(sliderContainerSize) {
```

Comment:
We can customize `getWidthAndHeightStyle` to take parameter optionally and then apply the width and height if passed. This this can be removed as well.